### PR TITLE
Js loops styling

### PR DESCRIPTION
**Pull request by mgczarnievicz**
Adding style to JS loops

### CODE REVIEW

**Review by JessicaLoers — APPROVED** [review]
**Nice** 👍
I have no complains.

File: sessions/js-loops/demo-end/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      hsl(17deg 100% 77%) 50%,
+      hsl(17deg 100% 72%) 51%,
+      hsl(16deg 100% 68%) 54%,
+      hsl(15deg 100% 64%) 58%,
+      hsl(15deg 100% 59%) 67%,
+      hsl(14deg 100% 53%) 99%);
+
+}
+
+body {
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-end/css/styles.css:45]:
```suggestion
body {
  max-width: 650px;
  margin-inline: auto;
```

File: sessions/js-loops/demo-end/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  margin: 0;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-end/css/styles.css:46]:
```suggestion
```

File: sessions/js-loops/demo-end/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  font-family: system-ui;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-end/css/styles.css:51]:
```suggestion
  min-height: 100dvh;
```

File: sessions/js-loops/demo-end/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  background: var(--primary);
+  background-image: var(--bg-gradient);
+}
+
+main {
+  width: 50%;
+  min-width: 300px;
+  padding: 10%;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-end/css/styles.css:59]:
```suggestion
  min-width: 300px;
  padding: 16px;
```

File: sessions/js-loops/demo-start/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      hsl(17deg 100% 72%) 51%,
+      hsl(16deg 100% 68%) 54%,
+      hsl(15deg 100% 64%) 58%,
+      hsl(15deg 100% 59%) 67%,
+      hsl(14deg 100% 53%) 99%);
+
+}
+
+body {
+  margin: 0;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-start/css/styles.css:46]:
```suggestion
  max-width: 650px;
  margin-inline: auto;
```

File: sessions/js-loops/demo-start/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  font-family: system-ui;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-start/css/styles.css:51]:
```suggestion
  min-height: 100dvh;
```

File: sessions/js-loops/demo-start/css/styles.css [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+  background: var(--primary);
+  background-image: var(--bg-gradient);
+}
+
+main {
+  width: 50%;
+  min-width: 300px;
+  padding: 10%;
```
Comment (F-Kirchhoff) [inline sessions/js-loops/demo-start/css/styles.css:59]:
```suggestion
  min-width: 300px;
  padding: 16px;
```

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Gimena,
I think its great to have some basic styling that we can reuse for all our challenges. Maybe we can export them to a "globals.css" file (for sessions after css structure of course) to indicate that they can be used in any other challenge. Further, I added some suggestions to the basic styling. I was wondering if one could simplify the background gradient to 3 or 4 lines to make the default styling less crowded? I think it would be great to have one "globals.css" file that can be used for every challenge and add additional styling in the "index.css" or somewhere else as needed for the specific elements.

**Comment by mgczarnievicz** [comment]
> Hey Gimena, I think its great to have some basic styling that we can reuse for all our challenges. Maybe we can export them to a "globals.css" file (for sessions after css structure of course) to indicate that they can be used in any other challenge. Further, I added some suggestions to the basic styling. I was wondering if one could simplify the background gradient to 3 or 4 lines to make the default styling less crowded? I think it would be great to have one "globals.css" file that can be used for every challenge and add additional styling in the "index.css" or somewhere else as needed for the specific elements.

Hey, totally, we can have a global one with all the variables. I will do it next and change it. But if it is okay with you could we merge it so the student can have it nice this afternoon?
Regarding the  gradient, I used a gradient generator form the internet and that was, what I was given. We can see and search to simplifying it.

**Pull request merged**